src/components/QRCodeReader.js

```diff
@@ -5,194 +5,197 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
```

Comment:
This screen was re-built using function components to be better able to use the new lib's hooks.